docs/src/components/IconsList.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import {
+  Button,
+  Card,
+  CardSection,
+  Coupon,
+  Heading,
+  InputField,
+  Stack,
+  Text,
+  Tile,
+} from "@kiwicom/orbit-components";
+import { Link as LinkIcon, Search } from "@kiwicom/orbit-components/icons";
+import iconsList from "@kiwicom/orbit-components/lib/data/icons.json";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import { CodeBlock } from "./Code";
+import { copyTimeout, pascalize } from "../utils/common";
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+  grid-gap: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+const StyledAnchorWrapper = styled.a`
+  a {
+    float: left;
+    padding-top: 4px;
+    padding-right: 2px;
+    margin-left: -22px;
+  }
+
+  svg {
+    height: 20px;
+    width: 20px;
+    visibility: hidden;
+  }
+
+  &:hover svg {
+    visibility: visible;
+  }
+`;
+
+interface IIconObject {
+  character: string;
+  description: string;
+  svg: string;
+  url: string;
+}
+
+interface IconProps {
+  iconName: string;
+  opened: boolean;
+  setOpenedIcon: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const Icon = ({ iconName, opened, setOpenedIcon }: IconProps) => {
+  const [copied, setCopied] = useState(false);
+  useEffect(() => {
+    copyTimeout(copied, setCopied);
+  }, [copied, setCopied]);
+
+  const iconObject: IIconObject = iconsList[iconName];
+  const pascalName = pascalize(iconName);
+
+  return (
+    <div id={iconName}>
```

Comment:
do we need `id` here? we are trying to avoid usage of IDs, only when there is specific reason like svg masks.

Reply:
Is there a better way to generate anchors on the page?